tests/device_tests/evolu/test_evolu_commands.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from trezorlib import evolu
+from trezorlib.debuglink import SessionDebugWrapper as Session
+
+# make -C core test_emu TESTOPTS="-k test_evolu_commands.py"
+
+pytestmark = pytest.mark.models("safe")
+
+
+@pytest.mark.models(skip="thp")
+def test_evolu_get_delegated_identity_is_constant(session: "Session"):
+    response = evolu.get_delegated_identity_key(session)
+    private_key = response.private_key
```

Comment:
changing the return value of `evolu.get_delegated_identity_key` to bytes (private key - directly) will allow you to do the following simplification
```suggestion
    private_key = evolu.get_delegated_identity_key(session)
```